app/routers/metrics.py

```diff
@@ -0,0 +1,87 @@
+from typing import Annotated, Literal
+import fastapi
+from pydantic import BaseModel, Field
+
+router = fastapi.APIRouter(
+    prefix="/metrics",
+    tags=["metrics"],
+    responses={404: {"description": "Not found"}},
+)
+
+
+async def metric_id_param(
+    metric_id: Annotated[
+        Literal["LossPersistence", "Coverage"],
+        fastapi.Path(description="metric you whish to query"),
+    ]
+):
+    return {"metric_id": metric_id}
+
+
+async def defined_areas_params(
+    area_type: Annotated[
+        str,
+        fastapi.Query(description="type of the predefined area", example="ea"),
+    ],
+    area_id: Annotated[
+        str, fastapi.Query(description="id of the area", example="CAR")
+    ],
+):
+    return {"area_type": area_type, "area_id": area_id}
+
+
+class Polygon(BaseModel):
+    polygon: str = Field(
+        description="GeoJSON polygon to determine the query area"
+        # TODO: Add input example
+    )
+
+
+# TODO: Successful response examples for /areas
+# TODO: 422 response examples for /areas and /layer
+
+
+@router.get("/{metric_id}/areas")
```

Comment:
Im getting this error

![imagen](https://github.com/PEM-Humboldt/biotablero-search-backend/assets/19979979/72384540-c8ca-4eda-a514-f8d6fdb6b842)

Reply:
http://localhost:8000/metrics/Coverage/areas?area_type=ea&area_id=CAR

Reply:
fixed